assemblies/prd-ce/src/main/resources/resource/system/osgi/log4j2.xml

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<Configuration status="WARN">
+   <Appenders>
+      <RollingFile name="PENTAHOFILE" fileName="logs/pdi.log"
+                   filePattern="logs/pdi.%d{yyyy-MM-dd}.log">
+         <PatternLayout>
+            <Pattern>%d %-5p [%c] %m%n</Pattern>
+         </PatternLayout>
+         <Policies>
+            <TimeBasedTriggeringPolicy />
+         </Policies>
+         <DefaultRolloverStrategy />
+      </RollingFile>
+      <Console name="PENTAHOCONSOLE">
```

Comment:
Missing Threshold filter set to INFO.  Isn't it weird that we see this set to different levels in different log4j.xml files =)